python/lancedb/table.py

```diff
@@ -524,22 +523,90 @@ def update(
 class LanceTable(Table):
     """
     A table in a LanceDB database.
+
+    This can be opened in two modes: if `version` is None then the latest version
+    is opened and can be tracked. If `version` is an integer then that version is
```

Comment:
It's not clear to me what `can be tracked` means.

Reply:
I've rephrased this a bit. It now says the table can be in one of two modes: standard, which tracks the latest version, and time-travel, which is fixed at an older version. Hopefully this is clearer.